src/php/index.php

```diff
@@ -26,5 +24,7 @@ function run(string $guide) {
         ->resolve();
 
     $app = $app(...$args);
+
+    $app->executeMigration();
```

Comment:
isn't that run by the guide?

Reply:
Shouldnt it be:
- automatically run when the guide is loaded (so that users can directly play with requests)
- maybe also exposed through \App\Playground to re-run migrations via UI if needed
?